guides/release/card-sdk/routing.md

```diff
@@ -1,5 +1,19 @@
-Routing changed a lot from v1 to v2 of the Cardstack Hub and the builder.
```

Comment:
TBH we should just remove the routing section. There is overlying routing approach new hub like there was in the old hub, and the routing that we have in cardhost is definitely not the thing we want (where the canonical URL of the card is encoded into the URL)